clang/test/dpct/cmake_migration/case_008/expected.txt

```diff
@@ -0,0 +1,17 @@
+find_package(IntelSYCL)
+dpct_find_package(CUDA REQUIRED)
+dpct_find_package(CUDA ${REQUIRED_CUDA_VERSION} REQUIRED)
+#Current YAML rule comments find_package(MPI) as it changes CMAKE_CXX_COMPILER
+#find_package(MPI)
+#find_package(MPI REQUIRED)
+#Current YAML rule comments find_package(OpenMP) as it changes CMAKE_CXX_COMPILER
+#find_package(OpenMP)
+#find_package(OpenMP REQUIRED)
+dpct_find_package(CUDAToolkit)
+dpct_find_package(CUDAToolkit 11.4 REQUIRED)
+dpct_find_package(CUDAToolkit REQUIRED cublas cufft nvToolsExt)
```

Comment:
After a further thought, cuda libraries "cublas cufft nvToolsExt" should not be appeared in sycl macro/functions, which will confuse users,  we should migrate these libraries' name in yaml based rules.